build.sbt

```diff
@@ -40,6 +40,7 @@ val elasticsearch5Version = "5.5.0"
 val featranVersion = "0.1.26"
 val gcsConnectorVersion = "1.6.3-hadoop2"
 val gcsVersion = "1.8.0"
+val grpcContextVersion = "1.2.0"
```

Comment:
Why are you using this version specifically ? It was published on March 2017. The latest version is 1.12.0 and was published on May 2018.

Reply:
@danielnorberg ping